### DISCUSSION TITLE
Meson issues while installing cfclient

### DISCUSSION THREAD

**Discussion by NickNavarrete** [bitcraze/discussions#1605] [Q&A] [answered]
Hello, I am trying to install cfclinet using the 'pip install cfclient command'. 
While running the command I get the following error

![image](https://github.com/user-attachments/assets/8beafdbb-4329-483f-89e3-9730165659fc)

I installed gcc on this machine as I assumed it was looking for a cpp compiler, but the error still persists. The error also occurs when running from the vscode terminal.

Has anyone encountered this issue? What is the solution?

**Comment by gemenerik** [comment]
Hi Nick. I assume you don't actually have command in the call, right? It should be `pip install cfclient`.
Are you on Windows 10 or 11? How did you install Python?
If you can next time copy the content of the terminal into your post/comment that makes it easier for us to read and help you debug.

**Reply by NickNavarrete** [reply]
Correct, I did not have 'command' in the call.
Windows 10, Python was installed using the installer from the python website, I am using python version 3.13.

I also have cflib in the same directory and it fails to install using the same 'pip install cflib' command for the same reason.

Thank you

**Reply by gemenerik** [reply]
Can you install other pip packages? For example something like `numpy`.

**Reply by NickNavarrete** [reply]
Yes I can install other pip packages, including less standard ones such as vicon_dssdk. 

Could the Python version be causing issues?

**Reply by gemenerik** [reply, marked as answer]
Please give Python 3.12 a try.

Windows 10 + Python 3.13 _should work_, but I’ll ensure we verify it on our end as well.

**Reply by gemenerik** [reply]
For Python 3.13 you most likely need [Visual Studio](https://visualstudio.microsoft.com/downloads/), select the Desktop Development with C++ workload in the installer. This is because you will need to compile some wheels.

**Reply by NickNavarrete** [reply]
Sorry for getting back to this so late, but switching to python 3.12 fixed the issue.
